Makefile

```diff
@@ -59,6 +59,9 @@ test: clean-test ## run tests quickly with the default Python
 test-all: clean-test ## run tests on every Python version with tox
 	tox
 
+integration-test:
+	CONTENTWORKSHOP_URL=https://hotfixes.studio.learningequality.org python tests/test_chef_integration.py
```

Comment:
Wondering if we want to test this against all three branches, but not a blocker.